README.md

```diff
@@ -58,7 +58,15 @@ To run a local dev instance, which gives you hot reload and a devtool ([Vue DevT
 
 ```bash
 npm run dev
-``` 
+```
+
+You must also start a MongoDB server locally. To do so, install MongoDB Server 4 from [the website](https://www.mongodb.com/download-center/community), and then run the following in another command line. This will run your local MongoDB server at http://localhost:27017:
```

Comment:
We can use a https://elements.heroku.com/addons/mongolab to run a dev instance. 

More complicated explanation I will send you internally